plugins/inputs/win_eventlog/win_eventlog.go

```diff
@@ -132,7 +132,7 @@ func (w *WinEventLog) Gather(acc telegraf.Accumulator) error {
 	if w.subscription == 0 {
 		w.subscription, err = w.evtSubscribe(w.EventlogName, w.Query)
 		if err != nil {
-			w.Log.Error("Subscription error:", err.Error())
+			return fmt.Errorf("Windows Event Log subscription error: %v", err.Error())
```

Comment:
Error strings should not be capitalized: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
I think Windows is a proper noun and should be capitalized

Reply:
You're right! My bad :)

Reply:
Not a problem :)